### PR TITLE
azureml containers repo update

### PR DESCRIPTION
**Pull request by v-tashish**

### CODE REVIEW

**Review by vizhur — APPROVED** [review]

File: base/gpu/openmpi4.1.0-cuda11.1-cudnn8-ubuntu18.04/release-notes.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
+**"Note: Support to Ubuntu 18.04 images will be dropped starting January 2023 as it reached EOL"**
+
 -  **Image Name**: docker pull mcr.microsoft.com/azureml/openmpi4.1.0-cuda11.1-cudnn8-ubuntu18.04 :
 
+:20221101.v1
+------------------- 
+- released images to mitigate vulnerabilities vulnerabilities.
```
Comment (elesel) [inline base/gpu/openmpi4.1.0-cuda11.1-cudnn8-ubuntu18.04/release-notes.md:7]:
```suggestion
- released images to mitigate vulnerabilities
```

File: base/gpu/openmpi4.1.0-cuda11.2-cudnn8-ubuntu20.04/release-notes.md [hunk 2]
```diff
@@ -1,5 +1,9 @@
 -  **Image Name**: docker pull mcr.microsoft.com/azureml/openmpi4.1.0-cuda11.2-cudnn8-ubuntu20.04 :
 
+:20221101.v1
+------------------- 
+- released images to mitigate vulnerabilities vulnerabilities.
```
Comment (elesel) [inline base/gpu/openmpi4.1.0-cuda11.2-cudnn8-ubuntu20.04/release-notes.md:5]:
```suggestion
- released images to mitigate vulnerabilities
```

File: base/gpu/openmpi4.1.0-cuda11.3-cudnn8-ubuntu20.04/release-notes.md [hunk 3]
```diff
@@ -1,5 +1,11 @@
 -  **Image Name**: docker pull mcr.microsoft.com/azureml/openmpi4.1.0-cuda11.3-cudnn8-ubuntu20.04 :
 
+:20221101.v1
+------------------- 
+-   [CVE-2021-43618](https://ubuntu.com/security/notices/USN-5672-1) 
+
+- released images to mitigate vulnerabilities vulnerabilities.
```
Comment (elesel) [inline base/gpu/openmpi4.1.0-cuda11.3-cudnn8-ubuntu20.04/release-notes.md:7]:
```suggestion
- released images to mitigate vulnerabilities
```

File: base/gpu/openmpi4.1.0-cuda11.6-cudnn8-ubuntu20.04/release-notes.md [hunk 4]
```diff
@@ -1,5 +1,9 @@
 -  **Image Name**: docker pull mcr.microsoft.com/azureml/openmpi4.1.0-cuda11.6-cudnn8-ubuntu20.04 :
 
+:20221101.v1
+------------------- 
+- released images to mitigate vulnerabilities vulnerabilities.
```
Comment (elesel) [inline base/gpu/openmpi4.1.0-cuda11.6-cudnn8-ubuntu20.04/release-notes.md:5]:
```suggestion
- released images to mitigate vulnerabilities
```

**Review by elesel — CHANGES_REQUESTED** [review]
This needs a little bit of grammar help.

File: base/gpu/openmpi4.1.0-cuda11.0.3-cudnn8-ubuntu18.04/release-notes.md [hunk 5]
```diff
@@ -1,5 +1,15 @@
+**"Note: Support to Ubuntu 18.04 images will be dropped starting January 2023 as it reached EOL"**
```
[the remark this answers is not in the document]
Comment (v-tashish) [reply base/gpu/openmpi4.1.0-cuda11.0.3-cudnn8-ubuntu18.04/release-notes.md]: done

File: base/gpu/openmpi4.1.0-cuda11.0.3-cudnn8-ubuntu18.04/release-notes.md [hunk 6]
```diff
@@ -1,5 +1,15 @@
...
+
 -  **Image Name**: docker pull mcr.microsoft.com/azureml/openmpi4.1.0-cuda11.0.3-cudnn8-ubuntu18.04 :
 
+:20221101.v1
+------------------- 
+-   **Dependencies** :
+-   [CVE-2018-16860](https://ubuntu.com/security/notices/USN-5675-1) 
+-   [CVE-2022-3515](https://ubuntu.com/security/notices/USN-5688-1)
+
+- released images to mitigate Heimdal,Libksba vulnerabilities.
```
Comment (elesel) [inline base/gpu/openmpi4.1.0-cuda11.0.3-cudnn8-ubuntu18.04/release-notes.md:11]: There really should be a space here:
```suggestion
- released images to mitigate Heimdal, Libksba vulnerabilities.
```

**Review by elesel — APPROVED** [review]

**Review by elesel — APPROVED** [review]

**Pull request merged**